src/main/java/duke/main/TaskList.java

```diff
@@ -0,0 +1,46 @@
+package duke.main;
+
+import duke.task.Task;
+import java.util.ArrayList;
+
+/**
+ * Contains the task list. Has operations to add and delete tasks in the list.
+ */
+public class TaskList {
+    protected ArrayList<Task> taskList;
```

Comment:
"tasks" might be a better variable name here to adhere to code style of having plural form for collections of objects